pkg/users/account-details.js

```diff
@@ -145,12 +145,11 @@ export function AccountDetails({ accounts, groups, current_user, user, shells })
                                     console.log("Account locked state doesn't match desired value, trying again.");
                                     // only retry once to avoid uncontrolled recursion
                                     change_locked(value, true);
-                                } else
-                                    set_edited_locked(null);
+                                }
                             });
                 })
                 .catch(error => {
-                    set_edited_locked(null);
+                    setEditedLocked(null);
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-e74b7696-20250721-143844-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)